packages/amplication-client/src/Application/constants.ts

```diff
@@ -0,0 +1,37 @@
+export const COLORS = [
+  {
+    value: "#F5B82E",
```

Comment:
Extract color codes to constants and use them for values and keys in later object.